script.js

```diff
@@ -0,0 +1,54 @@
+const form = document.querySelector('#form');
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const bookList = document.getElementById('books-list');
+let collection = JSON.parse(localStorage.getItem('collection')) || [];
+let book = [{
+  title: 'Alpen',
+  author: 'Zalpen',
+  id: 1234574,
+}];
```

Comment:
[OPTIONAL] It would be nicer to remove this useless variable to keep your code clean.